docs/rules.md

```diff
@@ -23,6 +23,7 @@ Runs the webpack-cli under bazel.
 | :------------- | :------------- | :------------- | :------------- | :------------- |
 | <a id="webpack_bundle-name"></a>name |  A unique name for this target.   | <a href="https://bazel.build/docs/build-ref.html#name">Name</a> | required |  |
 | <a id="webpack_bundle-args"></a>args |  Command line arguments to pass to Webpack.<br><br>These argument passed on the command line before arguments that are added by the rule. Run <code>bazel</code> with <code>--subcommands</code> to see what Webpack CLI command line was invoked.<br><br>See the [Webpack CLI docs](https://webpack.js.org/api/cli/) for a complete list of supported arguments.   | List of strings | optional | [] |
+| <a id="webpack_bundle-data"></a>data |  Runtime dependencies to include in binaries/tests that depend on this library.<br><br>        If this list contains linked npm packages, npm package store targets or other targets that provide 'JsInfo',         'NpmPackageStoreInfo' providers are gathered from 'JsInfo'. This is done directly from 'npm_package_stores' and         'transitive_npm_package_stores' fields of these and for linked npm package targets, from the underlying         npm_package_store target(s) that back the links via 'npm_linked_packages' and 'transitive_npm_linked_packages'.<br><br>        Gathered 'NpmPackageStoreInfo' providers are used downstream as direct dependencies when linking a downstream         'npm_package' target with 'npm_link_package'.   | <a href="https://bazel.build/docs/build-ref.html#labels">List of labels</a> | optional | [] |
```

Comment:
backticks like the other PR?

Reply:
yea, I'm going to regex and fix them all. I blame vscode.

Reply:
fixed